client/src/analytics/events.ts

```diff
@@ -48,4 +48,5 @@ export enum EVENTS {
   EXPLORER_ADD_GENE_AND_SELECT_HISTOGRAM = "EXPLORER_ADD_GENE_AND_SELECT_HISTOGRAM",
   EXPLORER_SELECT_HISTOGRAM = "EXPLORER_SELECT_HISTOGRAM",
   EXPLORER_GENE_INFO_BUTTON_CLICKED = "EXPLORER_GENE_INFO_BUTTON_CLICKED",
+  WMG_CLICK_NAV = "WMG_CLICK_NAV",
```

Comment:
We might need to prefix this event with `EXPLORER_`, so Amanda can tell whether the click is from Explorer or Data Portal. Or if she doesn't care then never mind!

Reply:
I asked - prefix not necessary as plausible tracks source URL